docs/security/trust/deploying_notary.md

```diff
@@ -0,0 +1,37 @@
+<!--[metadata]>
++++
+title = "Deploying Notary"
+description = "Deploying Notary"
+keywords = ["trust, security, notary, deployment"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Deploying Notary
+
+## The easy way
```

Comment:
Because there's no "other" way currently, I think we can remove this heading; this is what it currently looks like;

<img width="735" alt="screen shot 2015-11-11 at 13 26 11" src="https://cloud.githubusercontent.com/assets/1804568/11091147/0cc4be42-8878-11e5-842a-ae2b6dd97ded.png">